packages/app-admin-ui/index.js

```diff
@@ -25,8 +26,8 @@ class AdminUIApp {
       throw new Error("Admin path cannot be the root path. Try; '/admin'");
     }
 
-    if (authStrategy && authStrategy.authType !== 'password') {
-      throw new Error('Keystone 5 Admin currently only supports the `PasswordAuthStrategy`');
+    if (authStrategy && authStrategy.authType !== 'password' && !authStrategy.loginPath) {
```

Comment:
Do you still need the check against the `password` auth type?

Reply:
Yeah because the password auth strategy doesn't have a `.loginPath` property. We don't require a password auth to have a loginPath to redirect to because this package (app-admin-ui) is itself able to handle login flow for password auth. 

So the requirement for an auth strategy is to (1) be password type (login web flow can be handled by package) **or** (2) have a loginPath (which handles login web flow).